pkg

```diff
@@ -34,8 +34,6 @@ cd $(readlink -m $0/..)
 SRCDIR=${SRCDIR-$(readlink -e ./sources)}
 SPEC=${SPEC:-vault.spec}
 sudo sed -i s/.centos// /etc/rpm/macros.dist
-# Workaround https://bugs.centos.org/view.php?id=12793
-sudo sed -i '/^baseurl/s/$releasever/7.7.1908/g' /etc/yum.repos.d/CentOS-Sources.repo
```

Comment:
Can you explain this hunk? It looks unrelated to other changes in the commit.

Reply:
According to the link in the link, this is a workaround related to yum-builddep. It is not used here, or it works. Anyway, I could build the packages on centos 7 and 8 without it, and I do not want to modify the configurations files of the centos 8 container with things related to centos 7

Reply:
Ok, fine. Next time, can you make that kind of change into a dedicated commit?

Reply:
ok, I'll do that next time. thanks for the review